package.json

```diff
@@ -29,7 +30,9 @@
     "prismjs": "^1.6.0",
     "prop-types": "^15.5.10",
     "react": "^15.5.4",
+    "react-autosuggest": "^9.3.1",
     "react-dom": "^15.5.4",
+    "react-instantsearch": "^4.0.7",
```

Comment:
react-instantsearch can be removed already. I find it odd that it wasn't removed when I did yarn remove